src/oil_reservoir_synthesizer/_oil_simulator.py

```diff
@@ -47,14 +47,14 @@ def __init__(self, ooip=2000, goip=2500, woip=2250):
         self._wells = {}
         self._bpr = {}  # Block pressure for each block
 
-    def addWell(
+    def add_well(
```

Comment:
Isn't this a breaking public API change? This symbol is referenced in the README, so I would assume so. Should there be a `addWell` forwarder function that also throws a `DeprecationWarning`?

Reply:
Added wrapper and DeprecationWarning.